openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -55,6 +55,34 @@
 # PRIVATE METHODS
 #=============================================================================================
 
+# Directory paths used by ForceField to discover offxml files.
+_installed_offxml_dir_paths = []
+
+def _get_installed_offxml_dir_paths():
+    """Return the list of directory paths where to search for offxml files.
+
+    This function load the information by calling all the entry points
+    registered in the "openff.forcefielddirs" group. Each entry point
+    (i.e. a function) should return a list of paths to directories
+    containing the offxml files.
+
+    Returns
+    -------
+    _installed_offxml_dir_paths : List[str]
```

Comment:
This shouldn't block the merge, just curious: What does a return value with an underscore indicate?

Reply:
Whops! This is a typo. Thanks for catching it. I'll fix it before merging.